### DISCUSSION TITLE
提议：增加视频播放倍率

### DISCUSSION THREAD

**Discussion by RichardHammondX** [Richasy/Bili.Uwp#815] [想法]
网页版bili可以通过“HTML5视频播放器增强脚本 https://greasyfork.org/scripts/381682 ” 这样的脚本来达到超过3倍速播放视频的目的，不知道Bili.Uwp从技术上实现类似功能是否困难，很希望加上这样的功能！

**Comment by Richasy** [comment]
技术上没困难，但是超越3倍速这个需求为什么会存在？

**Reply by RichardHammondX** [reply]
很高兴得到你的回复！需求场景可以是 1.网课：某个视频内容水分多但需了解，存在字幕的情况下高倍速可以提高效率。 2.娱乐视频：带字幕的情况下高倍速观看可以节约大量时间。 3.测评类视频：可以快速掠过自己不关心的部分去看后面的内容。 4.生活记录类视频： 此类视频内容较为轻松但时长较长，语速和进度推进均较为缓慢，假如高倍速播放可以节约大量时间

**Reply by Richasy** [reply]
我可以尝试在之后将其作为一个可选功能加入，你预期的最高倍率是多少呢？

**Reply by RichardHammondX** [reply]
目前大量视频是在7-14分钟的区间内的，按照10分钟视频长按哔哩手机客户端3倍速播放某些视频还是略感拖沓的情况下，个人实测下来上最高不超过6倍，我自己用网页版的时候一般是调到3~4.5倍看的。

**Reply by RichardHammondX** [reply]
在倍速的控制上，个人感受是按键盘数字键直接更改倍率的用户体验是很不错的

**Reply by Richasy** [reply]
了解，谢谢反馈，我会在之后的版本中尝试提高快进速率上限

**Reply by RichardHammondX** [reply]
感谢！！

**Reply by Richasy** [reply]
不客气，代码已经提交：[播放速率功能调整 #843](https://github.com/Richasy/Bili.Uwp/pull/843)，将包含在下一次更新中。到时候你需要先在设置中打开 `播放速率增强`

**Reply by RichardHammondX** [reply]
thx a lot！
